Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\authView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
Internal Server Error: /orders/
Traceback (most recent call last):
  File "D:\PY\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py", line 86, in post
    print(request.session.data())
          ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'SessionStore' object has no attribute 'data'
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Not Found: /favicon.ico
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
Internal Server Error: /orders/
Traceback (most recent call last):
  File "D:\PY\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py", line 93, in post
    return render(request, self.template_name, {'role': role, 'form': form, 'orders': None})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\defaulttags.py", line 326, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 710, in resolve
    obj = self.var.resolve(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 842, in resolve
    value = self._resolve_lookup(context)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\template\base.py", line 909, in _resolve_lookup
    current = current()
              ^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\utils.py", line 91, in as_table
    return self.render(self.template_name_table)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\utils.py", line 54, in render
    context = context or self.get_context()
                         ^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 225, in get_context
    top_errors = self.non_field_errors().copy()
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 252, in non_field_errors
    return self.errors.get(
           ^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 192, in errors
    self.full_clean()
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 327, in full_clean
    self._clean_fields()
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 334, in _clean_fields
    value = bf.initial if field.disabled else bf.data
                                              ^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\boundfield.py", line 136, in data
    return self.form._widget_data_value(self.field.widget, self.html_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\forms.py", line 216, in _widget_data_value
    return widget.value_from_datadict(self.data, self.files, html_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\PY\Lib\site-packages\django\forms\widgets.py", line 730, in value_from_datadict
    getter = data.get
             ^^^^^^^^
AttributeError: 'OrderForm' object has no attribute 'get'
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\bookView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
D:\IGI\253502_KRASYOV_11\IGI\LR5\bookshop\app\views\orderView.py changed, reloading.
Watching for file changes with StatReloader
matplotlib data path: D:\PY\Lib\site-packages\matplotlib\mpl-data
CONFIGDIR=C:\Users\misti\.matplotlib
interactive is False
platform is win32
CACHEDIR=C:\Users\misti\.matplotlib
Using fontManager instance from C:\Users\misti\.matplotlib\fontlist-v330.json
